src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+promise1.then(result => printMessage(result));
+promise2.catch(error => printMessage(error));
+
+function printMessage(message) {
+  const messageForPrint = document.createElement('div');
+
+  messageForPrint.innerText = message;
+
+  message === 'Promise was resolved!'
+    ? messageForPrint.classList.add('message')
+    : messageForPrint.classList.add('error-message');
```

Comment:
better not rely on message text
you could pass additional parameter to `printMessage` (boolean error for example with default to false)